Makefile

```diff
@@ -0,0 +1,17 @@
+# Include the library makefile
+include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
+	golang.mk \
+	lib/tmp.mk \
+	targets/openshift/codegen.mk \
+	targets/openshift/controller-gen.mk \
```

Comment:
We don't need these 2 targets for this project

Reply:
Removed!